server/app/models/fusor/deployment.rb

```diff
@@ -17,7 +17,6 @@ class Deployment < ActiveRecord::Base
 
     validates :name, :presence => true, :uniqueness => {:scope => :organization_id}
     validates :organization_id, :presence => true
-    validates :lifecycle_environment_id, :presence => true
 
```

Comment:
I also have this change in my PR at: https://github.com/fusor/fusor/pull/98/files#diff-547dd7c2c8e7c70a3e1e1688795d462bL19